docs/concepts-broker-load-balancing-types.md

```diff
@@ -6,55 +6,38 @@ sidebar_label: "Types"
-Split balance|Verify if the extensible broker load balancer can decrease the time of automatic bundle splitting and balance the overloaded topics.| Measure the time of splitting 1 bundle into 128 bundles and balancing the load (10,000 topics/publishers) when hot-spot topics have high loads.|15 mins|13 mins
+| Dimension                                  | Modular broker load balancer                                                                                                                                                                                                                                                                  |Extensible broker load balancer
+|--------------------------------------------|-----------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|---
+ Available version                          | Pulsar 1.7.0 <br/> <br/>2017                                                                                                                                                                                                                                                                  |Pulsar 3.0.0 <br/> <br/>2023
+ Load Balance Metadata Store and Replication | Dependent on ZooKeeper for load balance metadata store and replication.<br/> <br/> - All broker load data and all bundle load data are stored in ZooKeeper and replicated to all brokers via ZooKeeper watchers. <br/><br/>- All bundle ownerships are stored in Zookeeper as ephemeral locks. | Dependent on system topics and table views for load balance metadata store and replication. <br/> <br/> - All broker load data are stored in a non-persistent system topic and replicated to all brokers via table views. <br/> <br/>- Each broker publishes only top k bundles' load data in a non-persistent system topic, and only the leader broker consumes it via a table view. <br/> <br/> - Bundle ownerships are stored in a persistent system topic and replicated to all brokers via table views.<br/> <br/> * The absolute size of the replicated load data and the complexity of the replication decreased due to passing the data through memory-only, non-persistent topics.
+ Load Balance Decision<br/>(Load Balance leader dependency)| Single leader broker decides on bundle-broker assignment, bundle splitting, and bundle unloading. <br/> <br/> * The leader broker requires global load information                                                                                                                    | Each broker decides and runs bundle-broker assignment(lookup) and bundle splitting based on the local(replicated) information. <br/><br/> The leader broker still decides on bundle unloading globally.
```

Comment:
```suggestion
 Load Balance Decision<br/>(Load Balance leader dependency)| Single leader broker decides on bundle-broker assignment, bundle splitting, and bundle unloading. <br/> <br/> - The leader broker requires global load information                                                                                                                    | Each broker decides and runs bundle-broker assignment (lookup) and bundle splitting based on the local (replicated) information. <br/><br/> The leader broker still decides on bundle unloading globally.
```

Reply:
I intentionally used * to note that the following is an annotation.

`The leader broker requires global load information `